client/js/editmode.js

```diff
@@ -1190,13 +1190,50 @@ onLoad(function() {
   on('#addSeat', 'click', async function() {
     const seats = widgetFilter(w=>w.get('type')=='seat');
     const maxIndex = Math.max(...seats.map(w=>w.get('index')));
-    await addWidgetLocal({
+    const id = await addWidgetLocal({
       type: 'seat',
       index: seats.length && maxIndex ? maxIndex+1 : 1,
       x: 840,
       y: 90
+    })
+    await addWidgetLocal({
+      id: id+'C',
```

Comment:
So this always adds the count, and it's up to the game designer to remove it in the editor? (Not necessarily requesting a change, just making sure I understand).

Reply:
Yes, that is the idea.  I don't know if that is best.  The only alternative I could come up with was to add a new property to seats (with a file updater of course) that had the count and let designers use that on a widget.  I decided I liked the current approach best.  But open to other ideas.

Reply:
Could also provide multiple options for the seat widget from the add widget screen.

Reply:
> Could also provide multiple options for the seat widget from the add widget screen.

Great idea.  Done.